app/components/billing-subscription.js

```diff
@@ -1,11 +1,29 @@
 import Component from '@ember/component';
-import { reads, and } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import { reads, and, not } from '@ember/object/computed';
+import { computed } from '@ember/object';
 
+const sourceToSentence = {
+  manual: 'This is a manual subscription.',
+  github: 'This subscription is managed by GitHub Marketplace.',
+  stripe: 'This plan is paid through Stripe.'
+};
```

Comment:
Generally texts should belong to templates - this is the most obvious place to look for them eventually. It's better to have `if`s chain there:
```hbs
{{#if (eq this.subscription.source 'manual')}}
  This is a manual subscription.
{{else if (eq this.subscription.source 'github'}}
  This subscription is managed by GitHub Marketplace.
{{else}}
  ...
{{/if}}
```